specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.0/keyvault.json

```diff
@@ -0,0 +1,4834 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "KeyVaultClient",
+    "description": "The key vault client performs cryptographic key operations and vault operations against the Key Vault service.",
+    "version": "7.0-preview"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{vaultBaseUrl}",
+    "useSchemePrefix": false,
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "name": "vaultBaseUrl",
+        "description": "The vault name, for example https://myvault.vault.azure.net.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/keys/{key-name}/create": {
+      "post": {
```

Comment:
Not all of the operations have examples, will they be added later?

Reply:
Yes.  This version (7.0-preview) of the spec is a copy of the 2016-10-01 spec, with a few breaking changes so far.  The 2016-10-01 didn't have examples for these operations.  We plan on adding examples for all before moving 7.0 to stable.